SmartBadge/lib/screen/icons.py

```diff
@@ -39,8 +39,8 @@ class Wall(lv.obj):
 
     def __init__(self, parent, width, height, x, y):
         super().__init__(parent)
-        self.set_size(width*5, height*4)
-        self.set_pos(x*5, y*4)
+        self.set_size(width*4, height*4)
```

Comment:
Do you want global constant? Don't mind if it's hard-coded given time constraints.